runtime/src/bank.rs

```diff
@@ -3340,6 +3440,7 @@ impl Bank {
     ) {
         let sanitized_txs = batch.sanitized_transactions();
         debug!("processing transactions: {}", sanitized_txs.len());
+        // TODO - this is the 1st metric
```

Comment:
not too clear what this TODO is addressing

Reply:
Ah, forgot to remove this comment. Added to "bookmark" the metric count I need for grafana. No real use.